gql loader not resolving and breaking the CLI

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
After publishing `0.5.0-alpha1` as part of #284 , and running either `yarn build` or `npm build`, there is a webpack error showing about not being able to process GraphQL files.
```shell
$ npm run build

> greenwood-getting-started@1.0.0 build /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started
> greenwood build

-------------------------------------------------------
Welcome to Greenwood ♻️
-------------------------------------------------------
Reading project config
Initializing project workspace contexts
Generating graph of workspace files...
Scaffolding out project files...
Generate pages from templates...
Writing imports for md...
Writing Lit routes...
setup index page and html
Scaffolding complete.
Building project for production.
Building SPA from compilation...
./node_modules/@greenwood/cli/src/data/queries/config.gql 1:6
Module parse failed: Unexpected token (1:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> query {
|   config {
|   	devServer {
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! greenwood-getting-started@1.0.0 build: `greenwood build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the greenwood-getting-started@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owenbuckley/.npm/_logs/2020-03-27T01_02_50_534Z-debug.log
```

## Details
This is likely due to our [**webpack** loader configuration](https://github.com/ProjectEvergreen/greenwood/blob/release/0.5.0/packages/cli/src/config/webpack.config.common.js#L137)
```js
      }, {
        test: /\.(graphql|gql)$/,
        exclude: /node_modules/,
        loader: 'graphql-tag/loader'
      }]
```

The `exclude` rule is actually excluding our own code.  I think we should just remove because this will likely block any other dependencies.

Also, we don't block _node_modules_ in any other loader configuration.  🤷‍♂

--- a/packages/cli/src/config/webpack.config.common.js
+++ b/packages/cli/src/config/webpack.config.common.js
@@ -135,7 +135,6 @@ module.exports = ({ config, context }) => {
         loader: 'file-loader'
       }, {
         test: /\.(graphql|gql)$/,
-        exclude: /node_modules/,
         loader: 'graphql-tag/loader'
       }]
     },

